test/probes/http-common.js

```diff
@@ -313,22 +305,18 @@ describe(`probes.${p}`, function () {
         res.end('done')
       })
 
-      let trasportXtrace = ''
-
       helper.doChecks(emitter, [
         function (msg) {
           // the edge in "Continuation" is from trace parent.
           expect(msg).property('Edge', otherTracestateOrgPart.slice(3).split('-')[0].toUpperCase())
           expect(msg).property('sw.tracestate_parent_id')
           expect(msg).property('sw.w3c.tracestate')
-
```

Comment:
The comment in line 310 says that edge should be from the traceparent, which is correct.  But the assertion after that seems to check the edge is from tracestate instead.  I'd exepct the `sw.tracestate_parent_id` property to capture our upstream span ID carried in tracestate, and also that `sw.w3c.tracestate` is only set if there was other vendor information in the tracestate header.

Reply:
The comment was wrong. 

Fixed.

Note:
* A *Flow* request is one that is directly downstream from a SolarWinds APM instrumented app. It has valid, `traceparent` and `tracestate` headers. Org <u>is in</u> `tracestate`. The span id part of `traceparent` <u>match</u> the span id part in `tracestate`. 
* A Continuation request is one that comes directly downstream form an Open Telemetry instrumented service, but has passed through a SolarWinds APM instrumented service in the past. It has valid `traceparent` and `tracestate` headers. Org <u>is in</u> `tracestate`. The span id part of `traceparent` <u>does not match</u> the span id part in `tracestate`.

The Edge always goes back to "our" span (if exist). It this case (Continuation) it goes back to the one before the OTEL chasm, which is stored in tracestate, thus creating the link over the chasm.

In this test headers are:
```
  traceparent: '00-11111111111111111111111111111111-7a71b110e5e3588d-01',
  tracestate: 'sw=9999888855667788-01',
```
The span id part of `traceparent` <u>does not match</u> the span id part in `tracestate` so the edge to "our" span is from tracestate.

Reply:
See edge defined in: https://my.dc-01.dev-ssp.solarwinds.com/138094000386806784/traces/5287C1AE1729E00EC45639067B9ECB0A/421F401A9A334CE7/graph#event-98E5B0ECE32AD8B5 for SolarWinds APM (S) linking over 3rd party OTEL chasm (X).

Reply:
The current implementation ins the correct implementation.

Reply:
See "standard OTel-generated segments" in: https://my.dc-01.dev-ssp.solarwinds.com/138094000386806784/traces/5005BCD29749B037E83ECC55F6CE9CC0/5CE3ABDA691741D9/details for SolarWinds APM (S) linking to vanilla OTEL based JavaScript Agent (O). (chain SOSSOSSOS)

<img width="1200" alt="Screen Shot 2022-06-09 at 1 40 48 PM" src="https://user-images.githubusercontent.com/891105/172940692-25bf271c-b3ab-4620-98ef-c34da2edb08f.png">